### PR TITLE
[incubator/fluentd-elasticsearch] Extended Readme and added OWNERS file

### PR DESCRIPTION
**Pull request by monotek**
**What this PR does / why we need it**:
Extended Readme and added OWNERS file

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/fluentd-elasticsearch/README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ $ helm install incubator/fluentd-elasticsearch
 ## Introduction
 
 This chart bootstraps a [Fluentd](https://www.fluentd.org/) daemonset on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+Its meant to be a drop in replacement for fluentd-gcp on GKE which sends logs to Googles Stackdriver service but can be also used anywhere else where Elasticsearch logbackend is needed.
```
Comment (timstoop) [inline incubator/fluentd-elasticsearch/README.md]: "It's mean to be" (as it's shorthand for "it is")
"... Google's Stackdriver service, but can also be used in other places where logging to ElasticSearch is required." (sounds better to me)

File: incubator/fluentd-elasticsearch/README.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ $ helm install incubator/fluentd-elasticsearch
...
+The used Docker image also contains Googles detect exeptions (for Java multiline stacktraces), Prometheus exporter, Kubernetes metadata filter & Systemd plugins.
```
Comment (timstoop) [inline incubator/fluentd-elasticsearch/README.md]: "Google's detect exceptions" (add the apostrophe and the c)

**Review by timstoop — CHANGES_REQUESTED** [review]

File: incubator/fluentd-elasticsearch/README.md (hunk 1 above)
Comment (monotek) [reply incubator/fluentd-elasticsearch/README.md]: done

File: incubator/fluentd-elasticsearch/README.md (hunk 2 above)
Comment (monotek) [reply incubator/fluentd-elasticsearch/README.md]: done

**Pull request merged**